seqr/views/utils/terra_api_utils.py

```diff
@@ -192,10 +194,55 @@ def user_get_workspace_acl(user, workspace_namespace, workspace_name):
               "canCompute": true
             }
           }
+
     """
     path = "api/workspaces/{0}/{1}/acl".format(workspace_namespace, workspace_name)
     try:
         return _user_anvil_call('get', path, user).get('acl', {})
+    # Exceptions are handled to avoid error when a non-anvil user trying to validate permissions on AnVIL
```

Comment:
but shouldn't that throw an error?

Reply:
I'm not sure we still need it or not. It is needed only when a user has been revoked from accessing a workspace and we still allow him to access other projects. If we don't handle these exceptions, the user won't open the dashboard (project table) properly.

Reply:
So if you look at all the upstream uses for this function, it is eventually used by the helper method for `get_json_for_project_collaborator_list` which is called in a) the project page data api to show all the details for the project page and b) after a project collaborator has been updated or deleted. The reason we catch the errors here is because if you can't get the collaborator list for any reason, we want the project page to show normally just without any collaborators, rather than blocking the user, which is good behavior.

 This comment really doesn't make that clear.  This has nothing to do with "non-anvil" users and its never used for permission validation. Please change the comment to actually describe why we are catching these errors, maybe "Exceptions are handled as being unable to display the collaborator list should not otherwise block seqr project access"